### PR TITLE
Changed OrderedSet from Vec to BoundedVec

### PR DESCRIPTION
**Pull request by brettkolodny**

### CODE REVIEW

**Review by xlc — COMMENTED** [review]
Need more tests for out of bound insert

File: oracle/src/lib.rs [hunk 1]
```diff
@@ -89,6 +89,8 @@ pub mod module {
 
 		/// Weight information for extrinsics in this module.
 		type WeightInfo: WeightInfo;
+
+		type MaxHasDispatchedSize: Get<u32>;
```
Comment (xlc) [inline oracle/src/lib.rs:93]: need a doc comment

File: utilities/src/ordered_set.rs [hunk 2]
```diff
@@ -60,74 +58,100 @@ impl<T: Ord> OrderedSet<T> {
...
+	fn from(v: BoundedVec<T, S>) -> Self {
 		Self::from(v)
 	}
 }
 
+#[cfg(feature = "std")]
+impl<T, S> fmt::Debug for OrderedSet<T, S>
+where
+	T: fmt::Debug,
+	S: Get<u32> + fmt::Debug,
```
Comment (xlc) [inline utilities/src/ordered_set.rs:75]: I don't think we need this line?

Comment (brettkolodny) [reply utilities/src/ordered_set.rs:75]: I can't seem to derive `Debug` and the code won't compile without it since it's needed for `assert` in the tests

Comment (xlc) [reply utilities/src/ordered_set.rs:75]: I mean is `S: Get<u32> + fmt::Debug,` required?

Comment (brettkolodny) [reply utilities/src/ordered_set.rs]: You're right that wasn't needed

**Pull request merged**